### DISCUSSION TITLE
Make theme toggle the ability to follow system scheme

### DISCUSSION THREAD

**Discussion by gouku** [facebook/docusaurus#7160] [Ideas]
I really like how the new [MDN](https://developer.mozilla.org/en-US/) handle themes:

<img width="211" alt="image" src="https://user-images.githubusercontent.com/9361308/162911687-564e8547-c5c9-4a58-857f-692650bc3905.png">

**Comment by slorber** [comment]
We don't provide this out of the box and don't really plan to, but you can swizzle `Navbar/ColorModeToggle` and add your own UX

Note we don't support `setColorMode(undefined)` to reset to system scheme, but that could be useful to support 👍

**Comment by pranabdas** [comment]
I think you can use following configuration to query and apply system color mode instead of a hardcoded default.
```
colorMode: {
          respectPrefersColorScheme: true,
},
```
https://docusaurus.io/docs/api/themes/configuration#respectPrefersColorScheme
